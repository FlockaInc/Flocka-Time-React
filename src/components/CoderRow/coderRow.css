.my-data {
  background-color: rgba(42, 159, 214, 0.4);
}