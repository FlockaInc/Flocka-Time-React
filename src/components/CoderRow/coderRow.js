import React from 'react';
import './coderRow.css';

function CoderRow(props) {
  let rowClass = props.currentUser ? 'my-data' : '';


  return (
    <tr className={rowClass}>
      <td>{props.rank}</td>
      <td>{props.username}</td>
      <td>{props.total}</td>
      <td>{props.dailyAvg}</td>
    </tr>
  );
}

export default CoderRow;