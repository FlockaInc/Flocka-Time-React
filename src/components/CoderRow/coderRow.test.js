import React from 'react';
import { render, cleanup } from '@testing-library/react';
import CoderRow from './coderRow';

afterEach(cleanup);

it('renders', () => {
  let table = document.createElement('table');
  let tbody = document.createElement('tbody');
  table.appendChild(tbody);

  const { asFragment, container } = render(<CoderRow rank={2} username={'test'} total={6.9} dailyAvg={1.2} currentUser={'test@test.com'} />, {
    container: document.body.appendChild(table)
  });

  expect(asFragment()).toMatchSnapshot();
});