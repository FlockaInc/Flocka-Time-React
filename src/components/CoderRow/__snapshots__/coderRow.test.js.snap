// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<DocumentFragment>
  <tr
    class="my-data"
  >
    <td>
      2
    </td>
    <td>
      test
    </td>
    <td>
      6.9
    </td>
    <td>
      1.2
    </td>
  </tr>
</DocumentFragment>
`;
